app/lti.go

```diff
@@ -9,16 +9,15 @@ import (
 	"errors"
 	"fmt"
 	"net/http"
+	"strconv"
 
 	"github.com/mattermost/mattermost-server/v6/app/request"
 	"github.com/mattermost/mattermost-server/v6/model"
 	"github.com/mattermost/mattermost-server/v6/shared/mlog"
 	"github.com/mattermost/mattermost-server/v6/store"
 )
 
-//
-//	GetLTISettings() reads the LTI Config from Plugin Config
-//
+// GetLTISettings() reads the LTI Config from Plugin Config
```

Comment:
I'd contend this change was unnecessary. We haven't settled on a style for function comments in go, and in some ways this was easier to read than other function comments that we've created. Since this function wasn't being modified in any other way maybe we should leave it along until we're ready to add function comments consistently